baseplate/lib/prometheus_metrics.py

```diff
@@ -438,3 +438,59 @@ def get_requests_total_metric(cls) -> Counter:
     @classmethod
     def get_active_requests_metric(cls) -> Gauge:
         return cls.active_requests
+
+
+class PrometheusSQLAlchemyClientMetrics:
+    prefix = "database_client"
+
+    labels = [
+        "statement",
+    ]
```

Comment:
I feel like there should be more labels than this. Like endpoint and stuff.  
What's statement exactly btw?